fs/layer/node.go

```diff
@@ -165,6 +176,7 @@ var _ = (fusefs.InodeEmbedder)((*node)(nil))
 var _ = (fusefs.NodeReaddirer)((*node)(nil))
 
 func (n *node) Readdir(ctx context.Context) (fusefs.DirStream, syscall.Errno) {
+	n.logOperation(context.Background(), "Readdir")
```

Comment:
Why doesn't it take the incoming context (`ctx`)?

Reply:
Updated to take the incoming context.